examples/rp/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 embassy-embedded-hal = { version = "0.1.0", path = "../../embassy-embedded-hal", features = ["defmt"] }
+embassy-hal-common = { version = "0.1.0", path = "../../embassy-hal-common", features = ["defmt"] }
```

Comment:
`embassy-hal-common` is for internal use only, no crate should depend on it except embassy hals.

Reply:
Removed